src/main/res/values/strings.xml

```diff
@@ -147,13 +145,13 @@ limitations under the License.
     <string name="activity_type_running">running</string>
     <string name="activity_type_sailing">sailing</string>
     <string name="activity_type_seaplane">seaplane</string>
-    <string name="activity_type_skate_boarding">skate boarding</string>
+    <string name="activity_type_skate_boarding">skateboarding</string>
     <string name="activity_type_skating">skating</string>
     <string name="activity_type_skiing">skiing</string>
     <string name="activity_type_sky_jumping">sky jumping</string>
     <string name="activity_type_sled">sled</string>
     <string name="activity_type_snow_boarding">snow boarding</string>
-    <string name="activity_type_snow_shoeing">snow shoeing</string>
+    <string name="activity_type_snow_shoeing">snowshoeing</string>
```

Comment:
Does this mess up the entries in settings.xml?

Reply:
Yes, it would as we store in the preferences directly (best guess we even store the localized string) as well as per track.
Seemingly, this was not abstracted away aka mixing up localization and data storage.

Reply:
Will it work if I change it in both places?

Reply:
Sadly it won't work; if it is wrong, just change it; the problem is not that big.